eventing/samples/gcp-pubsub-source/README.md

```diff
@@ -52,17 +52,18 @@ source is most useful as a bridge from other GCP services, such as
       gcloud iam service-accounts keys create knative-source.json \
         --iam-account=knative-source@$PROJECT_ID.iam.gserviceaccount.com
       ```
-   1. Create a secret on the kubernetes cluster for the downloaded key. You need
-      to store this key in `key.json` in a secret named `gcppubsub-source-key`
-
+   1. Create two secrets on the kubernetes cluster with the downloaded key.
```

Comment:
I can change it to:

Create two secrets on the kubernetes cluster with the downloaded key stored in the secrets' `key.json` key.

Is that better? 

My personal view is that because the lines just after the shell script mention that `key.json` is a value configured elsewhere, we don't need to call it out specifically in this preamble.

Reply:
Yeah, that makes sense on the preamble. Suggesting an edit on your last iteration, to make it less repetitive.

```suggestion
   1. Create two secrets on the Kubernetes cluster with the downloaded key stored in the `key.json` file.
```